configuration/config-txt/overclocking.md

```diff
@@ -29,7 +29,15 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 | sdram_freq_min | Minimum value of `sdram_freq` used for dynamic frequency clocking. The default value is `400`. |
 | over_voltage_min | Minimum value of `over_voltage` used for dynamic frequency clocking. The default value is `0`. |
 | temp_limit | Overheat protection. This sets the clocks and voltages to default when the SoC reaches this value in Celsius. Setting this higher than the default voids your warranty. The default value is `85`. |
-| current_limit_override | Disables SMPS current limit protection when set to `0x5A000020`. It requires this unusual value to ensure that it is not triggered accidentally. This can help if you are hitting a reboot failure when specifying a value for overclocking that is too high. For more information [see here](http://www.raspberrypi.org/phpBB3/viewtopic.php?f=29&t=6201&start=325#p170793). Changing this option may set the warranty bit. |
+
+This table describes the overvoltage settings for the various Pi versions.
+
+| Version | Default Overvoltage | Setting |
```

Comment:
Perhaps "Setting" should be "Effective Default Setting" (or words to that effect), since AIUI you can't manually apply a setting of 4.5 in `config.txt`?